examples/dcutr/README.md

```diff
@@ -0,0 +1,33 @@
+## Description
+
+The "Direct Connection Upgrade through Relay" (DCUTR) protocol allows peers in a peer-to-peer network to establish direct connections with each other, bypassing the need for relays and signaling servers. This example provides a basic usage of this protocol in **libp2p**.
```

Comment:
Not sure if "bypassing" is a good wording because we use relays in the dcutr protocol for signalling :)

Reply:
@thomaseizinger Just removed this part, this was useless and maybe prone to misunderstanding :)